tests/test-config.js

```diff
@@ -0,0 +1,5 @@
+/*!
+ * Copyright 2023 Digital Bazaar, Inc. All Rights Reserved
```

Comment:
Is this copyright declaration appropriate for content of a W3C repository? Seems to me, it should at least be _mutually_ held by DB and W3C, and probably the W3C line should not include "All Rights Reserved", but whatever CC or similar declaration...

Reply:
Good question on copyright...I'll do some digging on code contribution policy for the W3C.

This should at least include the license declaration for the BSD-3-Clause. I'll take the action to clean these up. Thanks @TallTed!

Reply:
@TallTed  Thanks for pointing that out. I've opened an issue here for this here https://github.com/w3c/vc-di-ecdsa-test-suite/issues/27.